src/models/tracepointformat.h

```diff
@@ -0,0 +1,35 @@
+/*
+    SPDX-FileCopyrightText: Lieven Hey <lieven.hey@kdab.com>
+    SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+
+    SPDX-License-Identifier: GPL-2.0-or-later
+*/
+
+#pragma once
+
+#include <QString>
+
+#include "data.h"
+
+class TracePointFormatter
```

Comment:
only used by `formatTracepoint`, so this can move to the `.cpp` file (into an anon namespace there?)